packer/install-dependencies.sh

```diff
@@ -19,6 +19,7 @@ if [[ ${FEATURE_FLAGS} == *"docker-engine"* ]]; then
     DOCKER_ENGINE_REPO="https://apt.dockerproject.org/repo"
     installDockerEngine
     installGPUDrivers
+    installContainerd
```

Comment:
Just to be conservative/iterative, let's keep the `installContainerd` invocation in the VHD for non-moby scenario.

(See below that it used to be here.)

I don't think we strictly need to be doing this installation, but the safest way is to make that change distinctly.

The reason we're removing it from moby is that the moby installation implementation currently installs containerd stuffs *itself*.